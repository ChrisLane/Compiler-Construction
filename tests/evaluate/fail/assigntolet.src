// This is to test that we cannot change the value of a let variable. This should not evaluate.
main() {
    let x = 50 {
        x = 10;
    };
}
