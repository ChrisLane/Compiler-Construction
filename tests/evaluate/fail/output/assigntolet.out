Fatal error: exception Failure("Could not find a variable to update.")
